homeassistant/components/miele/sensor.py

```diff
@@ -770,23 +770,16 @@ class MielePhaseSensor(MieleSensor):
     @property
     def native_value(self) -> StateType:
         """Return the state of the sensor."""
-        ret_val = STATE_PROGRAM_PHASE.get(self.device.device_type, {}).get(
-            self.device.state_program_phase
+        return (
+            PROGRAM_PHASE[self.device.device_type](self.device.state_program_phase).name
+            if self.device.device_type in PROGRAM_PHASE
+            else None
         )
-        if ret_val is None:
-            _LOGGER.debug(
-                "Unknown program phase: %s on device type: %s",
-                self.device.state_program_phase,
-                self.device.device_type,
-            )
```

Comment:
Why did we remove the log? Is it because we rely on prints in `MieleEnum`?

Reply:
Yes